src/jcstress/java/reactor/pool/SimpleDequePoolStressTest.java

```diff
@@ -34,10 +34,10 @@
 public class SimpleDequePoolStressTest {
 
 	@JCStressTest
-	@Outcome(id = {"ACQUIRED '200', 0"}, expect = ACCEPTABLE,  desc = "acquired")
-	@Outcome(id = {"NOT ACQUIRED, 1"}, expect = ACCEPTABLE,  desc = "acquired failed fast during/before destroy")
-	@Outcome(id = {"ACQUIRED '1', 0"}, expect = FORBIDDEN,  desc = "acquired the first resource, before destroy")
-	@Outcome(id = {"ACQUIRED '1234', 0"}, expect = FORBIDDEN,  desc = "acquired the first resource, after destroy")
+	@Outcome(id = "200, 0", expect = ACCEPTABLE,  desc = "acquired a new resource")
+	@Outcome(id = "404, 1", expect = ACCEPTABLE,  desc = "acquired failed fast during/before destroy")
+	@Outcome(id = "1, 0", expect = FORBIDDEN,  desc = "acquired the first resource, before destroy")
+	@Outcome(id = "1234, 0", expect = FORBIDDEN,  desc = "acquired the first resource, after destroy")
```

Comment:
I have a personal distaste for magic numbers here, and prefer to use round/special numbers where I can, for example: 1, 100, 200 for good cases here, and -1 for bad case. But then again, that's a personal preference.

Reply:
that is a fair point, will try to strike a balance with the magic numbers so that they're kept simple while having state changes reflected as large gaps (so that there's no ambiguity in which state change led to the final value)